types/google-apps-script/apis/driveactivity_v2.d.ts

```diff
@@ -77,18 +77,26 @@ declare namespace GoogleAppsScript {
                 timestamp?: string | undefined;
             }
             interface DriveItem {
+                // file is deprecated; please use the driveFile instead.
                 file?: any;
+                // folder is deprecated; please use the driveFolder instead.
                 folder?: DriveActivity.Schema.Folder | undefined;
                 mimeType?: string | undefined;
                 name?: string | undefined;
                 owner?: DriveActivity.Schema.Owner | undefined;
                 title?: string | undefined;
+                driveFile?: any;
```

Comment:
Should be an object and not any; perhaps also add a File to Schema?

Reply:
The existing type for `file` here is `any`, so I'm not sure that it actually is a downgrade, but I personally agree that a stronger type would be good.